include/aspect/simulator_signals.h

```diff
@@ -250,6 +250,16 @@ namespace aspect
                                   const unsigned int compositional_index,
                                   const SolverControl &solver_control)> post_advection_solver;
 
+    /**
+    * A signal that is fired when the nonlinear solver scheme is done.
+    * Parameters are a reference to the SimulatorAccess, and an 
+    * information object that contains information
+    * about the final state (failure/success), number of 
+    * iterations and history of residuals.
+    */
+    boost::signals2::signal<void (const SimulatorAccess<dim> &,
```

Comment:
We normally don't pass SimulatorAccess to the signals because there are other ways to get access to it. Maybe remove?